modules/utxo-lib/test/bitgo/psbt/fromHalfSigned.ts

```diff
@@ -95,6 +96,18 @@ function runTest(
 
     function signPsbt(startTx: UtxoTransaction<bigint>, signers: BIP32Interface[]) {
       const psbt = UtxoPsbt.fromTransaction(startTx, prevOutputs);
+      psbt.updateGlobal({
+        globalXpub: walletKeys.triple.map((bip32) => {
+          const masterFingerprint = Buffer.alloc(4);
+          masterFingerprint.writeUInt32BE(bip32.parentFingerprint);
+          const extendedPubkey = bip32.neutered().toBase58();
+          return {
+            extendedPubkey: bs58check.decode(extendedPubkey),
+            masterFingerprint,
+            path: 'm',
```

Comment:
quick question here. Is this the only place where `globalXpub` get updated?

Reply:
xpubs are added to the psbt during the build route in WP. I was thinking of making this a function on `UtxoPsbt`, maybe this should be done now with this PR